Assets/Mediapipe/Samples/Common/Scripts/ImageSource/WebCamSource.cs

```diff
@@ -105,7 +105,23 @@ public override ResolutionStruct[] availableResolutions
 
     public override bool isPrepared => webCamTexture != null;
     public override bool isPlaying => webCamTexture != null && webCamTexture.isPlaying;
-    private bool _isInitialized;
+    private bool _isInitialized;
+
+    private void Awake()
+    {
+      if (overrideDefaultResolution == true)
+      {
+        //Flipped around if on mobile

```

Comment:
Insert a space character after `//`.

```suggestion
        // Flipped around if on mobile
```

The same goes for the other parts.